wiki/Storyboard/Scripting/en.md

```diff
@@ -1,34 +1,39 @@
 
-- Basic concepts:
-  - [General rules of storyboarding](General_Rules)
-  - [Objects](Objects)
-  - [Basic commands](Commands)
-  - [Compound commands](Compound_Commands)
-  - [Variables](Variables)
-- Extra:
-  - [Playing audio](Audio)
-  - [Storyboard scripting cheat sheet](Cheat_Sheet)
-  - [osu! file toggles](osu!_File_Toggles)
-  - [Short versions of commands](Shorthand)
+*Warning: Avoid editing storyboard scripts of a [beatmap](/wiki/Beatmap) currently open in the [beatmap editor](/wiki/Client/Beatmap_editor). Your work may be overwritten when saving or exiting the editor.*
 
-## Overview
+## Concepts and file format
 
-There is an overview of the storyboard scripting concepts provided by [Doomsday](https://osu.ppy.sh/users/18983): [osu! - Storyboard Scripting for Beginners (12:22)](https://www.youtube.com/watch?v=UJ1YLDs-bZg).
+<!-- storyboarding concepts should be at the same level as this article, and only details of scripting and the file format itself should be below this article, but sub-articles currently mix together this information -->
 
-## Full specification
+See the following articles:
 
-*Note: Save your work in the built-in editor first, as it optimises some things out, and only then in a plain text editor. **Save often** to prevent accidental progress loss.*
+- [General rules for storyboarding](General_Rules)
+- [Objects](Objects)
+- [Commands](Commands)
+- [Compound commands](Compound_Commands)
+- [Audio samples](Audio)
+- [osu! file toggles](osu!_File_Toggles)
+- [Variables](Variables)
+- [Shorthand](Shorthand)
+- [Cheat sheet](Cheat_Sheet)
 
-The storyboard scripting process is primarily based on [the official specifications](https://osu.ppy.sh/community/forums/topics/1869) and experience in making large storyboards. Although the osu! wiki articles provide simple examples, they are not step-by-step guides. Instead, the explanations aim to exhaustively detail how the basic scripting commands translate into the on-screen actions in osu!.
+<!-- (in a new section) seems like the place to link the top level of a storyboarding guide on the wiki... if there was one! -->
 
-Before writing storyboard scripts, check [the terms and basic concepts](/wiki/Storyboard/Scripting/General_Rules) to prevent possible further issues and misunderstanding. For more in-depth explanations, see:
+## Community tools
 
-- [Storyboard objects](/wiki/Storyboard/Scripting/Objects)
-- [Storyboard scripting commands](/wiki/Storyboard/Scripting/Commands)
+Various tools have been made by the community to abstract and build upon storyboard scripting, such as ![][flag_FR] [Damnae](https://osu.ppy.sh/users/989377)'s [storybrew](https://github.com/Damnae/storybrew "GitHub repository").
```

Comment:
i've listed more tools in [#4827](https://github.com/ppy/osu-wiki/issues/4827#issuecomment-990741545), but i suppose this can be ok to put here for now, although it does make this section very short

Reply:
oh nice. i'll list the ones that are general purpose & intended for newer storyboarders here, rest can be for some other article

Reply:
per last comment im gonna keep this as-is, but leave a note about what could be here if stuff from #4827 is added later